sympy/integrals/transforms.py

```diff
@@ -1008,9 +1008,34 @@ def repl(ex, *args):
 def _laplace_transform(f, t, s_, simplify=True):
     """ The backend function for Laplace transforms. """
     from sympy import (re, Max, exp, pi, Min, periodic_argument as arg_,
-                       arg, cos, Wild, symbols, polar_lift)
+                       arg, cos, Wild, symbols, polar_lift, limit,
+                       Add, DiracDelta)
```

Comment:
`Add` should now be removed.

Reply:
Oh sorry, I ran all tests and failed to notice that flake8 complained.